main.go

```diff
@@ -149,14 +158,23 @@ func main() {
 	var hosts []host.Host
 	var dhts []*dht.IpfsDHT
 	uniqpeers := make(map[peer.ID]struct{})
-	fmt.Fprintf(os.Stderr, "Running %d DHT Instances...", *many)
+	fmt.Fprintf(os.Stderr, "Running %d DHT Instances...\n", *many)
 	for i := 0; i < *many; i++ {
-		h, d, err := makeAndStartNode(ds, "/ip4/0.0.0.0/tcp/0", *relay)
+		var err error
+		for r := 0; r < *retries; r++ {
+			var h host.Host
+			var d *dht.IpfsDHT
+			h, d, err = makeAndStartNode(ds, "/ip4/0.0.0.0/tcp/0", *relay, *retries)
```

Comment:
Does creating the node itself (not just bootstrapping) fail? That should *never* happen.

Reply:
Well, I haven't seen that happen, I guess erroring/panicking here is more correct